Add rule docs for 8.2 rule changes (#1863)
elastic/security-docs: docs/detections/prebuilt-rules/rule-details/signed-proxy-execution-via-ms-workfolders.asciidoc

[[signed-proxy-execution-via-ms-workfolders]]

> === Signed Proxy Execution via MS WorkFolders

Reviewer: ```suggestion
=== Signed Proxy Execution via MS Work Folders
```

Identifies the use of Windows Work Folders to execute a potentially masqueraded control.exe file in the current working directory. Misuse of Windows Work Folders could indicate malicious activity.

*Rule type*: eql

*Rule indices*:

* winlogbeat-*
* logs-windows.*

*Severity*: medium

*Risk score*: 47

*Runs every*: 5 minutes

*Searches indices from*: now-9m ({ref}/common-options.html#date-math[Date Math format], see also <<rule-schedule, `Additional look-back time`>>)

*Maximum alerts per execution*: 100

*References*:

* https://docs.microsoft.com/en-us/windows-server/storage/work-folders/work-folders-overview
* https://twitter.com/ElliotKillick/status/1449812843772227588
* https://lolbas-project.github.io/lolbas/Binaries/WorkFolders/

*Tags*:

* Elastic
* Host
* Windows
* Threat Detection
* Defense Evasion

*Version*: 1

*Added ({stack} release)*: 8.2.0

*Rule authors*: Elastic, Austin Songer

*Rule license*: Elastic License v2

==== Investigation guide

[source,markdown]
----------------------------------
## Triage and analysis

> ### Investigating Signed Proxy Execution via MS WorkFolders

Reviewer: ```suggestion
### Investigating Signed Proxy Execution via MS Work Folders
```

Work Folders is a role service for file servers running Windows Server that provides a consistent way for users to access
their work files from their PCs and devices. This allows users to store work files and access them from anywhere. When
called, Work Folders will automatically execute any Portable Executable (PE) named `control.exe` as an argument before
accessing the synced share.

Using Work Folders to execute a masqueraded control.exe could allow an adversary to bypass application controls and
increase privileges.
 
#### Possible investigation steps

- Investigate the process tree starting with parent process WorkFolders.exe and child process control.exe to determine
if other child processes spawned during execution.

> - Trace the activity related to the `control.exe` binary to identify any continuing intrusion activity on the host.

Reviewer: For consistency, removing backticks for instances where we're not highlighting the name itself.
```suggestion
- Trace the activity related to the control.exe binary to identify any continuing intrusion activity on the host.
```

- Examine the location of the WorkFolders.exe binary to determine if it was copied to the location of the control.exe
binary. It resides in the System32 directory by default.
- Review the control.exe binary executed with Work Folders to determine maliciousness such as additional host activity
or network traffic.
- Determine if control.exe was synced to sync share, indicating potential lateral movement.
- Review how control.exe was originally delivered on the host, such as emailed, downloaded from the web, or written to
disk from a separate binary.
 
### False positive analysis 

- Windows Work Folders are used legitimately by end users and administrators for file sharing and syncing but not in the

> instance where a suspicious `control.exe` is passed as an argument.

Reviewer: ```suggestion
instance where a suspicious control.exe is passed as an argument.
```

### Response and remediation 

- Initiate the incident response process based on the outcome of the triage.
- Isolate the involved host to prevent further post-compromise behavior.

> - Review the Work Folders synced share to determine if the 'control.exe' was shared and if so remove it.

Reviewer: ```suggestion
- Review the Work Folders synced share to determine if the control.exe was shared and if so remove it.
```

> - If no lateral movement was identified during investigation, take the effected host offline if possible and remove the

Reviewer: ```suggestion
- If no lateral movement was identified during investigation, take the affected host offline if possible and remove the
```
